### DISCUSSION TITLE
Help Wanted from the Prisma team members 🙏

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [prisma/prisma#11029] [Q&A]
![Create Prisma Generator Banner](https://raw.githubusercontent.com/YassinEldeeb/create-prisma-generator/main/images/cool-banner.png)

So I've been developing [`create-prisma-generator`](https://github.com/YassinEldeeb/create-prisma-generator) a CLI like [`create-react-app`](https://github.com/facebook/create-react-app) for quite some time now to help the community with creating their own prisma generators by providing the smoothest getting started experience.

This is considered the **first part of my initiative** as [discussed here](https://github.com/prisma/prisma/discussions/10721) with @nikolasburk , but I need some help from Prisma team members to take a look at the getting started boilerplate, prisma SDK reference and let me know about any improvements that could be made cause my goal is to make it the standard for building community Prisma Generators.

**The second part of my initiative:** Also if anyone has a nice beginner friendly Prisma Generator project idea for something repetitive that can take advantage of the prisma schema definations, **let me know 🙏** cause I would like to build it in a dev.to blog as a guide for developers to open their mind to think in a different way when developing prisma generators.

### Get started:
```sh
npx create-prisma-generator
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
